src/UI.js

```diff
@@ -0,0 +1,162 @@
+          </button>
+          
+          <p class="card-text"></p>
+          <div class="d-flex justify-content-between">
+            
+            <button type="button" class="btn btn-primary comments-btn">
+              Comments
+            </button>
+
+            <div class="ml-3 btn btn-primary reservations-btn">Reservations</div>
+          </div>
+        </div>
+      </div>
+      `;
+    });
+
+    this.movie.innerHTML = output;
+
+    const movieKids = this.movie.children;
+
+    const openPop = document.getElementById('overlay');
+    const closePop = document.getElementsByClassName('close-pop');
+    const movieName = document.getElementsByClassName('movie-name');
+    const movieImage = document.getElementsByClassName('movie-image');
+    const movieYear = document.getElementsByClassName('movie-year');
+    const movieDesc = document.getElementsByClassName('description');
+    const commentAuthor = document.querySelector('#author-input');
+    const commentBody = document.querySelector('#comment-input');
+
+    function openCommentsModal(sourceId) {
+      console.log('works');
+      closePop.addEventListener('click', () => {
+        if (!openPop.classList.contains('hidden')) {
+          console.log('ok');
+          openPop.addClass(' hidden');
+        }
+      });
+      const imageM = document.querySelector(`#${sourceId}`)[0].getAttribute('src');
+      movieImage.src = imageM;
+
+      const nameM = document.querySelector(`#${sourceId}`)[1];
+      movieName.innerHTML = nameM[0].innerHTML;
+
+      const premierM = document.querySelector(`#${sourceId}`)[1];
+      movieYear.innerHTML = premierM[1].innerHTML;
+    }
+
+    Array.from(movieKids).forEach((item) => {
+      item.querySelector('.like-btn').addEventListener('click', (e) => {
+        // console.log(e.target.parentNode.parentNode.parentNode.id);
+        // console.log(item.id);
+        addLike(e.target.parentNode.parentNode.parentNode.id);
+        let addCount = parseInt(e.target.innerHTML, 10);
+        // addCount.toI
+        addCount += 1;
+        console.log(typeof addCount);
+        e.target.innerHTML = addCount;
+      });
+
+      item.querySelector('.comments-btn').addEventListener('click', (e) => {
+        console.log('works');
+        console.log(e.target.parentNode.parentNode.parentNode.id);
+        // remove class hidden
+        if (openPop.classList.contains('hidden')) {
+          openPop.classList.remove('hidden');
+          console.log('works fine');
+        }
+        // document.getElementById('overlay').removeClass('hidden');
+        // changeState.removeClass('hidden');
+        openCommentsModal(e.target.parentNode.parentNode.parentNode.id);
+      });
+
+      item.querySelector('.reservations-btn').addEventListener('click', () => {
+        console.log('works');
+        console.log(item.id);
+        // changeState.classList.remove('hidden');
+        // openReservationsModal(item.id);
+      });
+    });
+  }
```

Comment:
This is an optional change for you, I saw that it's only one person that works on this project and it's optional but you went ahead to implement the feature so I'm making this optional for you

- [ ] when the reservation button is clicked nothing happens